stable_baselines3/common/policies.py

```diff
@@ -87,6 +87,9 @@ def __init__(
 
         self.features_extractor_class = features_extractor_class
         self.features_extractor_kwargs = features_extractor_kwargs
+        # Automatically deactivate dtype and bounds checks
+        if normalize_images is False and features_extractor_class in {NatureCNN, CombinedExtractor}:
+            self.features_extractor_kwargs.update(dict(normalized_image=True))
```

Comment:
I'm not sure to understand this.
Is that to force normalization in `NatureCNN` and `CombinedExtractor`, even though it is disabled by the user?

Reply:
Oh, I got it. I read "normalize_images" instead of "normalized_image".

Reply:
yes... the idea is to have good defaults for the user and throw informative error when some explicit parameters are missing